app/models/organisation/organisation_type_concern.rb

```diff
@@ -31,8 +31,10 @@ module Organisation::OrganisationTypeConcern
     scope :hmcts_tribunals, -> {
       hmcts_id = Organisation.unscoped.where(slug: "hm-courts-and-tribunals-service").ids.first
       joins(:parent_organisational_relationships).
-        where(organisation_type_key: :tribunal_ndpb).
-        where("organisational_relationships.parent_organisation_id" => hmcts_id)
+      where("organisations.organisation_type_key = ? OR
```

Comment:
I think you might be able to do something like this to simplify the query.

```suggestion
      where(organisations: { organisation_type_key: %w(tribunal_ndpb tribunal) })
```

Reply:
I'll see if that works but it doesn't matter much as I'll delete all references to `tribunal_ndpb` after merging all my PRs and migrating orgs